bolt-contracts/src/interfaces/IBoltChallenger.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+interface IBoltChallenger {
+    enum ChallengeStatus {
+        Open,
+        Defended,
+        Lost
+    }
+
+    struct Challenge {
+        bytes32 id;
+        uint48 openedAt;
+        ChallengeStatus status;
+        address challenger;
+        address target;
```

Comment:
Perhaps some docs on these fields could be helpful -- I haven't very clear who is the `target` for example

Reply:
thanks - changed to `commitmentSigner` - this is the party that signed the user's commitment aka the operator